frontend/pages/register.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Steps, Step } from '../lib/step-form';
+import {
+  CreateAccountForm,
+  SelectRoleForm,
+  SelectPreferencesForm,
+  FillInformationForm,
+} from '../components/register';
+
+import Footer from '../components/Footer';
+
+const Register = () => {
+  return (
+    <div>
+      <div className="bg-gradient-to-tl from-secondary to-primary min-h-screen flex justify-center items-center">

```

Comment:
ใช้  `bg-gradient-blue-purple` แทน `bg-gradient-to-tl from-secondary to-primary min-h-screen` สิ มี customize ไว้ใน globals.css แล้ว

Reply:
อ้อ ทีแรกนึกว่า `bg-gradient-blue-purple` คือแบบยังไม่ to tl